src/storage/src/hummock/file_cache/mod.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod alloc;
+pub mod error;
+pub mod file;
+pub mod manager;
+
+async fn asyncify<F, T>(f: F) -> error::Result<T>
```

Comment:
Why not just call `spawn_blocking` directly? This seems opaque to me.

https://github.com/singularity-data/risingwave/blob/d3dc4dea4ca8cbc31db885045c642288c6211f73/src/storage/src/hummock/file_cache/file.rs#L128